src/01-forEach.js

```diff
@@ -12,7 +12,11 @@ const exampleSongData = require("../data/songs");
  * Logs out all of the song titles.
  * @param {Object[]} songs - An array of songs. See the song data for more.
  */
-function printAllSongTitles(songs) {}
+function printAllSongTitles(songs) {
+  songs.forEach((song) => {
+    console.log(`${song.title}`)
+  })
+}
```

Comment:
For this one (and the last one), you could simplify your console.log to just `console.log(song.title)`. No need to interpret JS within a string using a template literal if there's no other characters in the string!

It works, though, is the important thing.